### ISSUE TITLE
Request: Add an option to disable text wrapping

### ISSUE THREAD

**Issue by Thereatra** [basyura/inkdrop-sidetoc#10]
If you have long headings or a lot of sub headings it quickly becomes hard to visually navigate the toc, see image sample below.

![image](https://user-images.githubusercontent.com/23458968/148109715-68aa8d97-eb8f-4f09-9be5-f4119e1e81a7.png)

I suggestion an option to disable text wrapping. This could be done by either by cropping off the title, see image below, or by adding a horizontal scroll bar to the toc.

![image](https://user-images.githubusercontent.com/23458968/148109974-e1945515-e766-4b3a-9d3c-9a73c4abf4cd.png)

**Comment by basyura** [comment]
You can change the style with styles.less. 
https://docs.inkdrop.app/manual/style-tweaks

```less
.sidetoc-pane li {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}
```

**Comment by Thereatra** [comment]
Thank you it works! 👍
Though I still think it would be great to have a toggle in the settings if possible,

**Comment by basyura** [comment]
If others also need this feature, I will implement it.

**Comment by basyura** [comment]
"others" is me.
I'm implementing.  → #11

**Comment by basyura** [comment]
released #11
